app/controllers/sites/new.js

```diff
@@ -112,7 +114,11 @@ export default class CreateSitesNewController extends Controller {
     await primaryContact.save();
     await secondaryContact.save();
     address.fullAddress = combineFullAddress(address);
-
+    if (this.savedMode === false) {
+      address.mode = 'Manually saved';
+    } else {
+      address.mode = 'Automatically saved';
```

Comment:
**Nitpick:** Imo the term "saved" is not the most appropriate here. It is the filling in (or creation) of the address data that is manual or automated, not the saving of the address data. I would be inclined to replace "saved" with something more informative in this context.

Note, I try to look at something like this from the perspective of the developer who will have to interpret this predicate and its possible values at some time in the future. For example, the developer who will write the query to list all addresses manually entered by the users. They will have to be able to determine that the `mode` predicate (which name can also be more informative imo) and its string values contain the information needed for that query. Appropriate naming can make quite a lot of difference in how easy that process is, especially if the person writing the query does not know the context of this PR/ticket or any discussions around it.